data-source/json/test_relationships.json

```diff
@@ -0,0 +1,174 @@
+                        "id": "edit-person",
+                        "type": "ListEditQuestion",
+                        "question": {
+                            "id": "edit-question",
+                            "type": "General",
+                            "title": "What is the name of the person?",
+                            "answers": [{
+                                    "id": "first-name",
+                                    "label": "First name",
+                                    "mandatory": true,
+                                    "type": "TextField"
+                                },
+                                {
+                                    "id": "last-name",
+                                    "label": "Last name",
+                                    "mandatory": true,
+                                    "type": "TextField"
+                                }
+                            ]
+                        }
+                    },
+                    "remove_block": {
+                        "id": "remove-person",
+                        "type": "ListRemoveQuestion",
+                        "question": {
+                            "id": "remove-question",
+                            "type": "General",
+                            "title": "Are you sure you want to remove this person?",
+                            "answers": [{
+                                "id": "remove-confirmation",
+                                "mandatory": true,
+                                "type": "Radio",
+                                "options": [{
+                                        "label": "Yes",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No",
+                                        "value": "No"
+                                    }
+                                ]
+                            }]
+                        }
+                    }
+                },
+                {
+                    "type": "RelationshipCollector",
+                    "id": "relationships",
+                    "title": "This will iterate over the people list, capturing the one way relationships.",
+                    "list": "people",
+                    "question": {
+                        "id": "relationship-question",
+                        "type": "General",
+                        "title": "Thinking of {first_person_name}, {second_person_name} is their <em>...</em>",
+                        "answers": [{
+                            "id": "relationship-answer",
+                            "mandatory": true,
+                            "type": "Relationship",
+                            "playback": "{second_person_name} is {first_person_name_possessive} <em>…</em>",
+                            "options": [{
```

Comment:
It would be good to get the full list of relationships in here, so when they're entered during a test we can represent a full household.

Reply:
The idea was to have the smallest possible schema to demonstrate the functionality.